Source/g_game.c

```diff
@@ -1634,6 +1641,25 @@ static void G_DoLoadGame(void)
     save_p += sizeof totalleveltimes;
   }
 
+  // restore MUSINFO music
+  if (save_p - savebuffer <= length - 8)
+  {
+    char lump[9];
+    int i;
+
+    memcpy(lump, save_p, 8);
```

Comment:
Now we need to make sure that `lump[8]` is `\0`.

Reply:
You could e.g. initialize `char lump[9] = {0};`.

Reply:
Yes, I know. I was distracted.

Reply:
No problem, looks good now!